source/index.md

```diff
@@ -492,22 +521,80 @@ POST https://gateway.clearhaus.com/authorizations
     <i>Optional</i> <br />
     A reference to an external object, such as an order number.
   </dd>
+
+  <!-- deprecated -->
   <dt>threed_secure[pares]</dt>
-  <dd>[:base64:] <br /> <i>Optional</i> <br /> See more information on <a target="_blank" href="http://docs.3dsecure.io">3Dsecure.io</a></dd>
+  <dd>
+    Deprecated! Please use <code>card[pares]</code>. <br />
+    [:base64:] <br />
+    See more information on <a target="_blank" href="http://docs.3dsecure.io">3Dsecure.io</a>.
```

Comment:
`s/more information on/more information at` for consistency. Ref. https://github.com/clearhaus/gateway-api-docs/pull/53/files#diff-eeb8bdec9c0eef052c0532fce07c4bacR402.